src/forges/GitLab/tags.jl

```diff
@@ -0,0 +1,16 @@
+@json struct Release
+    tag_name::String
+    description::String
+end
+
+@json struct Tag
+    commit::Commit
+    release::Release
+    name::String
+    target::String
+    message::Union{Nothing, String}
```

Comment:
```suggestion
    message::String
```
The field automatically becomes nullable via the `@json` macro.